packages/typegpu-color/src/oklab.ts

```diff
@@ -0,0 +1,346 @@
+import tgpu from 'typegpu';
+import { f32, struct, vec3f } from 'typegpu/data';
+import { clamp, length, max, min, mix, select } from 'typegpu/std';
+
+const cbrt = tgpu['~unstable'].fn(
+  [f32],
+  f32,
+)(/* wgsl */ `(x: f32) -> f32 {
+  return sign(x) * pow(abs(x), 1.0 / 3);
+}`);
```

Comment:
Could we use the template string variant here? It gets treated by out formatter a lot nicer 💜
```suggestion
const cbrt = tgpu['~unstable'].fn([f32], f32) /* wgsl */`(x: f32) -> f32 {
  return sign(x) * pow(abs(x), 1.0 / 3);
}`;
```

Reply:
I'd use TGSL when `sign` is merged, right?

Reply:
What do you think about adding this to `std`? I know it's not in WGSL, but it is in `Math`.

Reply:
I think it's a good idea! Sounds useful 💜
Would you like to add it to std as part of this PR?

Reply:
It can be separate I think.